py-kms/templates/clients.html

```diff
@@ -95,9 +95,9 @@
         <p>Whoops?</p>
     </div>
     <div class="message-body">
-        This page seems to be empty, because no clients are available. Try to use the server with a compartible client to add it to the database.
+        This page appears empty as there are no available clients. To populate it, utilise a server equipped with a compatible client to add the necessary data to the database.
```

Comment:
```suggestion
        This page appears empty as there are no available clients. To populate it, utilise a compatible KMS client to add the necessary data to the database.
```

Reply:
This suggestion makes the error more clear as it does not need to be a "server" to make requests to the KMS server, both py-kms client and the Windows KMS client in Windows Server and Desktop are compatible for most versions of Windows.